cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -173,11 +205,54 @@ func generateRenewalFunction(cert *certsphase.KubeadmCert, caCert *certsphase.Ku
 	}
 }
 
-func generateRenewalCommand(cert *certsphase.KubeadmCert, cfg *renewConfig) *cobra.Command {
+func generateEmbeddedCertRenewalFunction(kdir, k string, cfg *renewConfig) func() {
+	return func() {
+		internalcfg, err := configutil.LoadOrDefaultInitConfiguration(cfg.cfgPath, &cfg.cfg)
+		kubeadmutil.CheckErr(err)
+
+		if cfg.useCSR {
```

Comment:
It would be nice to document the logic flow here on renewal of certs, as it's not clear why we are checking cfg's CSR.

Reply:
done